git/diff.go

```diff
@@ -9,9 +9,14 @@ import (
 	"github.com/waigani/diffparser"
 )
 
-func gitDiff(dir string) (string, error) {
+func gitDiff(dir string, isPR bool) (string, error) {
 	var out bytes.Buffer
-	cmd := exec.Command("git", "diff", "-G", "(^[^#])", "--staged")
+	var cmd *exec.Cmd
+	if isPR {
```

Comment:
isPR doesn't seem like the correct condition. `gromit git diff` has no connection to PRs. I think it makes more sense to combine the staged and unstaged diffs into one.

Reply:
You need a named commit, which I've implemented in my pr.